packages/components/pages/f-contact-preferences/stories/ContactPreferences.stories.js

```diff
@@ -72,11 +68,12 @@ export const ContactPreferencesComponent = () => ({
 
     template: '<contact-preferences ' +
     ':authToken="authToken" ' +
+    ':isAuthFinished="true" ' +
     ':locale="locale" ' +
-    ':smart-gateway-base-url="smartGatewayBaseUrl" ' +
+    'smart-gateway-base-url="" ' +
     ':prepareStory="prepareStory" ' +
     // eslint-disable-next-line no-template-curly-in-string
-    ' :key="`${authToken},${locale},${smartGatewayBaseUrl}`" />'
+    ' :key="`${authToken},${locale}`" />'
```

Comment:
```suggestion
    ':key="`${authToken},${locale}`" />'
```

Reply:
I think it was always like that do didn't want to break anything but can remove 👍